### PR TITLE
Env lookup

### PR DESCRIPTION
**Pull request by louy755**

### CODE REVIEW

File: config/initializers/client_environment.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ClientEnvironment
+
+  $ENVIRONMENTS = YAML.load_file("#{Rails.root}/config/client.yml")
```
Comment (ssharba) [inline config/initializers/client_environment.rb:3]: what is the structure of this yml file. can you share it here

Comment (louy755) [reply config/initializers/client_environment.rb:3]: @samer the config/client.yml!

**Review by louy755 — COMMENTED** [review]
@samer the config/client.yml??

File: config/initializers/client_environment.rb (hunk 1 above)
Comment (ssharba) [reply config/initializers/client_environment.rb:3]: - :name: Environment_1
  :env_id: 1
  :database_name: DB1
  :clients:
    - :client_id: 1
      :name: client_1
    - : client_id: 2
      :name: client_2
    - : client_id: 3
      :name: client_3
- :name: Environment_2
  :env_id: 2
  :database_name: DB2
  :clients:
    - : client_id: 4
      :name: client_4
    - : client_id: 5
      :name: client_3

Comment (louy755) [reply config/initializers/client_environment.rb:3]: proceeding!

File: lib/find_environment.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class FindEnvironment
+  class << self
+    def environment_for(client_id)
+      environment = $ENVIRONMENTS.detect do |client_environment|
```
Comment (ssharba) [inline lib/find_environment.rb]: let's use find instead of detect.

**Review by ssharba — CHANGES_REQUESTED** [review]

File: lib/find_environment.rb (hunk 2 above)
Comment (ssharba) [reply lib/find_environment.rb]: @MwMaki use this method in the logger please.

Comment (louy755) [reply lib/find_environment.rb]: Sure!

**Review by ssharba — APPROVED** [review]

**Pull request merged**